.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request
+description: Suggest an idea for vtk.js.
+title: ''
+labels: 'type: feature request 💡'
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        💡 If you do want to suggest a change or request a feature, please follow the template below to increase the chances of your report being looked at.
+
+        🧱 If adequate, assign one of the `Module: XXX` labels to your issue when creating it.
+  - type: textarea
+    id: description
+    attributes:
+      label: Needs
+      description: A clear and concise description of what need or problem you're trying to address.
+    validations:
+      required: true
+  - type: textarea
+    id: feature
```

Comment:
Not a big fan of the categories "Needs" and "Feature Description" when requesting changes or new features.  Perhaps:
"Request" (description) and "Motivation"
"Request" (description) and "Justification"
"Request" (description) and Reason"